migrations/20240914103655_add_full_name_to_users.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
```

Comment:
```suggestion
exports.up = function (knex) {
```

We don't need those comment lines